.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+version: 2
+
+main: &main
+    machine: true
+    environment:
+      # Used by qthelpers to close widgets after a defined time
+      - TEST_CI: "True"
+      - TEST_CI_APP: "True"
+    steps:
+      - checkout
+      - run:
+          command: docker pull dorowu/ubuntu-desktop-lxde-vnc:trusty
+      - run:
+          name: Install system packages
+          command: |
+            sudo apt-get update
+            sudo apt-get install libegl1-mesa
+      - run:
+          name: Install Miniconda
+          command: |
+            wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+            bash miniconda.sh -b -p $HOME/miniconda
+            export PATH="$HOME/miniconda/bin:$PATH"
+            conda install python=$PYTHON_VERSION
+      - run:
+          name: Install Spyder from GitHub
```

Comment:
There's no need to have this step now. It was needed before we released Spyder 3.2, but not anymore, so please remove it.

Reply:
@ccordoba12 I am not sure I understand what you are saying. Do you mean that it is easier to install spyder via conda instead of via github? That is true, but I think it is better to test against the version at the head of the 3.x branch on git, in order to catch any regressions. I know that 3.x is not supposed to introduce any backward incompatibilities, but it is good to test for it. Of course, this does mean testing against a moving target which has its own problems, but I felt this approach was the most useful.

Reply:
> I know that 3.x is not supposed to introduce any backward incompatibilities, but it is good to test for it

Ok. I was thinking to test against the latest release, but sure, we can test against the HEAD of 3.x

Then please move this to `install.sh` too.